### PR TITLE
Trigger Test Upload

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33546

## Description
Added a script to trigger the upload flow on a test env.
Merged the two upload packs to production scripts.
**The following flows had been tested:**
- [commit](https://app.circleci.com/pipelines/github/demisto/content/69826/workflows/e0a4a0c1-510e-42f4-a907-87eecf3d22b3)
- [nightly](https://app.circleci.com/pipelines/github/demisto/content/69706/workflows/85e5f02f-8726-4800-aab1-0683e20c984d) 
- [force upload - test env](https://app.circleci.com/pipelines/github/demisto/content/69581/workflows/84106cea-7963-4119-a56c-ce084f1b7a7b)
- [upload pack - test env
](https://app.circleci.com/pipelines/github/demisto/content/69564/workflows/8defa6ae-0ad2-4f74-a695-9c66dc22ec2c)

Need to test the prod upload flow.

### Confluence page
https://confluence.paloaltonetworks.com/display/DemistoContent/Trigger+a+test+upload+flow

### CODE REVIEW

**Review by guyfreund — CHANGES_REQUESTED** [review]
Looking good! 👏 , few notes:
1. Please output the `GCS_MARKET_BUCKET` & `SLACK_CHANNEL` values when we output the build params (in your branch in `config.yml` it should be line 240)

**Comment by Bargenish** [comment]
@guyfreund done
Screen Shot 2021-02-24 at 10 50 14

File: .circleci/config.yml [hunk 1]
```diff
@@ -1020,9 +1040,10 @@ jobs:
             echo $GCS_MARKET_KEY > $GCS_PATH
 
             GCS_BUILD_BUCKET="marketplace-ci-build"
-            GCS_MARKET_BUCKET="marketplace-dist"
-
-            python3 ./Tests/Marketplace/copy_and_upload_packs.py -a $PACK_ARTIFACTS -e $EXTRACT_FOLDER -pb "$GCS_MARKET_BUCKET" -bb "$GCS_BUILD_BUCKET" -s $GCS_PATH -n $CREATE_INSTANCES_JOB_NUMBER -c $CIRCLE_BRANCH -p "${PACKS_TO_UPLOAD}"
+            if [[ $GCS_MARKET_BUCKET != "marketplace-dist" ]]; then
+              STORAGE_BATH_PATH="upload-flow/builds/$CIRCLE_BRANCH/$CREATE_INSTANCES_JOB_NUMBER/content/packs"
```
[the remark this answers is not in the document]
Comment (Bargenish) [reply .circleci/config.yml:1044]: Should be `storage base path` (copied the name from the path branch).
The path to which upload the content to.

File: Tests/Marketplace/copy_and_upload_packs.py [hunk 2]
```diff
@@ -309,6 +309,10 @@ def main():
     if production_base_path:
         GCPConfig.STORAGE_BASE_PATH = production_base_path
```
[the remark this answers is not in the document]
Comment (guyfreund) [reply Tests/Marketplace/copy_and_upload_packs.py:312]: This is fine, we can change the value in line 308 to `content/packs` just to avoid confusion.

**Review by yaakovi — APPROVED** [review]

**Review by guyfreund — APPROVED** [review]

**Pull request merged**